internal/venice-common/src/main/java/com/linkedin/venice/utils/RegionUtils.java

```diff
@@ -44,4 +48,17 @@ public static String getRegionSpecificMetricPrefix(String localRegionName, Strin
       return localRegionName + "_from_" + regionName + "_remote";
     }
   }
+
+  /**
+   * A helper function to split a region list with {@link #REGION_FILTER_LIST_SEPARATOR}
+   */
+  public static Set<String> parseRegionsFilterList(String regionsFilterList) {
+    if (StringUtils.isEmpty(regionsFilterList)) {
+      return Collections.emptySet();
+    }
+    Set<String> regions = new HashSet<>();
+    String[] tokens = regionsFilterList.trim().toLowerCase().split(REGION_FILTER_LIST_SEPARATOR);
```

Comment:
can do 

`return  Stream.of(regionsFilterList.trim().split("\\s*,\\s*")).collect(Collectors.toSet());`

Reply:
We still support java 8 so cool syntax like this doesn't compile :(